### PR TITLE
Update the website roadmap

### PR DESCRIPTION
**Pull request by kellylee-gds**
Following Trang's preso at mid cycle demo, here's a proposed update to the roadmap on the website. It doesn't include any of the TBC items discussed where we're dependent on other teams.

### CODE REVIEW

File: src/community/roadmap/index.md [hunk 1]
```diff
@@ -33,18 +33,21 @@ We've also:
 We're:
 
 - investigating the [Navigation pattern](https://github.com/alphagov/govuk-design-system-backlog/issues/76)
-- making the guidance around form design and validation easier to find
-- working out how the Design System might need to change when used for pages embedded in apps using web views
-- starting to plan Design System Day 2024
+- planning Design System Day 2024 events
+- making it easier for teams using GOV.UK Frontend to only include the components a service is using
```
Comment (36degrees) [inline src/community/roadmap/index.md:37]: I think this probably needs to be removed as we've wrapped work on this now!

File: src/community/roadmap/index.md [hunk 2]
```diff
@@ -33,18 +33,21 @@ We've also:
...
 
 ## Coming up next
 
 We're getting ready to:
 
-- improve the way we organise technical documentation on the website
+- understand how we can better help users find what they need on the GOV.UK Design System website
+- ensure the GOV.UK Design System website is compliant with WCAG 2.2
+- improve the JavaScript of the Chararcter count, Tabs and Accordion components
```
Comment (36degrees) [inline src/community/roadmap/index.md:45]:
```suggestion
- improve the JavaScript of the Character count, Tabs and Accordion components
```

File: src/community/roadmap/index.md [hunk 3]
```diff
@@ -33,18 +33,21 @@ We've also:
...
 
 ## Future plans
 
 We plan to:
 
 - [drop support for Ruby Sass and LibSass](https://github.com/alphagov/govuk-frontend/issues/2637) and [migrate to the Sass module system](https://github.com/alphagov/govuk-frontend/issues/1791)
+- explore how the [accessibile autocomplete component](https://github.com/alphagov/accessible-autocomplete) could be added to the GOV.UK Design System
```
Comment (36degrees) [inline src/community/roadmap/index.md:52]:
```suggestion
- explore how the [accessible autocomplete component](https://github.com/alphagov/accessible-autocomplete) could be added to the GOV.UK Design System
```

**Comment by owenatgov** [comment]
It looks like this is now out of date. We've at the very least started the website WCAG 2.2 work.

**Comment by trang-erskine** [comment]
@owenatgov I missed this one needing a review a while back. It looks like the roadmap has been updated on 9 August already. Is it easier to close this and start again when we need to update the roadmap? I'm going to make a note of updating the roadmap after each cycle.

**Comment by owenatgov** [comment]
I'm jsut gonna close this since it's gotten forgotten. The roadmap does need updating but I don't know what should go on there instead.

**Pull request closed without merging**